docs/vc-spec.md

```diff
@@ -156,19 +155,22 @@ type ArgumentValue = variant { "Int" : int32; String : text };
 
 Specifically, the issuer checks via `signed_id_alias.credential_jws` that user identified by its `sub` claim on the
 issuer side has a valid `has_id_alias` principal for the purpose of attribute sharing, and that the credential
-described by `credential_spec` does apply to the `caller`.  When these checks are successful, the issuer
-prepares and returns a context in `PreparedCredentialData.prepared_context` (if any).  The returned prepared context is then
+described by `credential_spec` does apply to the `caller`. When these checks are successful, the issuer
+prepares and returns a context in `PreparedCredentialData.prepared_context` (if any). The returned prepared context is then
 passed back to the issuer in a subsequent `get_credential`-call (see below).
 This call must be authenticated, i.e. the sender must match the principal for which the credential is requested.
 
 **NOTE:**
 The value of `prepared_context` is basically used to transfer information between `prepare_credential`
 and `get_credential` steps, and it is totally up to the issuer to decide on the content of
-that field.  That is, the issuer creates `prepared_context`, and is the only entity that
-consumes it.  For example, when using [canister signatures](https://internetcomputer.org/docs/current/references/ic-interface-spec#canister-signatures)
+that field. That is, the issuer creates `prepared_context`, and is the only entity that
+consumes it. For example, when using [canister signatures](https://internetcomputer.org/docs/current/references/ic-interface-spec#canister-signatures)
 the context contains a time-stamped yet unsigned VC, for which the canister signature will be
 available only at `get_credential`-call.
 
+**NOTE:**
```

Comment:
This is the only actual change in this document. The rest are formatting changes.